src/main/java/robot/subsystems/drivetrain/pure_pursuit/Constants.java

```diff
@@ -0,0 +1,12 @@
+package robot.subsystems.drivetrain.pure_pursuit;
+
+public class Constants {
+    public final static double WEIGHT_SMOOTH = 0.85;
+    public final static double WEIGHT_DATA = 1-WEIGHT_SMOOTH;
+    public final static double TOLERANCE = 0.001;
+    public final static double LOOKAHEAD_DISTANCE = 0.3;
+    public final static double TRACK_WIDTH = 3;
+    public final static double Kv = 0;
+    public final static double Ka = 0;
```

Comment:
You should change this constant name into `kA`, as `Ka` is not in [Camel case](https://en.wikipedia.org/wiki/Camel_case).
```suggestion
    public final static double kA = 0;
```